src/main/java/com/minecolonies/coremod/commands/RandomTeleportCommand.java

```diff
@@ -91,7 +91,9 @@ public void execute(@NotNull MinecraftServer server, @NotNull ICommandSender sen
             return;
         }
         playerToTeleport.getCommandSenderEntity().addChatMessage(new TextComponentString("Buckle up buttercup, this ain't no joy ride!!!"));
+
         teleportPlayer(sender, playerToTeleport);
+        playerToTeleport.fallDistance=-5;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Assign this magic number 5 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)